pkgs/os-specific/darwin/apple-sdk/default.nix

```diff
@@ -1,10 +1,15 @@
 { stdenv, fetchurl, xar, cpio, pkgs, python, pbzx, lib }:
 
+let version = "10.12"; in
+
+# Ensure appleSdkVersion is up to date.
+assert stdenv.isDarwin -> stdenv.appleSdkVersion == version;
```

Comment:
Why don't we just use `stdenv.appleSdkVersion`?

Reply:
The attribute is macOS specific, without the condition evaluating darwin only packages on linux fails on the assertion instead of meta.platforms with a nice message.

Reply:
What I was trying to ask is: if `version` must be equal to `stdenv.appleSdkVersion`, then why is one of them not initialized from the other, as in `version = stdenv.appleSdkVersion or "invalid platform";`?